app/models/cautionary_alert.rb

```diff
@@ -3,20 +3,12 @@
 class CautionaryAlert
   include ActiveModel::Model
 
-  attr_accessor :propertyReference, :alerts
+  attr_accessor :locationAlert, :personAlert
```

Comment:
how about use plural here?

Reply:
Yeah I think that would make sense but the only reason I didn't do this is because of HACT naming. I'm not sure whether it is supposed to be pluralised based on the data model sheet. But happy to pluralise if this is compliant with HACT - i'll need to double check